### PR TITLE
feat: remove confirmation and show toast on goal delete

### LINKED ISSUE #2054: Replace 'are you sure' modal with Toast for deletion
**Is your feature request related to a problem? Please describe.**
It's very annoying when you're testing to have to click the 'don't show this again' every time you refresh the app state.

**Describe the solution you'd like**
Just show a Toast with message  
"Moved to Trash!  
We'll delete it in 7 days."

**Describe alternatives you've considered**
None.

**Additional context**
image

### PR DESCRIPTION
**Pull request by harshalranjhani**
Removed the confirmation modal when user tries to delete a goal. Instead it shows a toast directly which says "Moved to trash! We'll delete it in 7 days."

Fixes #2054 

Screenshot:
image

### CODE REVIEW

**Review by tijlleenders — DISMISSED** [review]
Thanks!

File: src/components/GoalsComponents/MyGoalActions/RegularGoalActions.tsx [hunk 1]
```diff
@@ -8,7 +8,7 @@ import ConfirmationModal from "@src/common/ConfirmationModal";
 import ZModal from "@src/common/ZModal";
 import archiveSound from "@assets/archive.mp3";
 
-import { lastAction, openDevMode, displayConfirmation } from "@src/store";
+import { lastAction, openDevMode, displayConfirmation, displayToast, displayToast } from "@src/store";
```
Comment (vinaybadgujar102) [inline src/components/GoalsComponents/MyGoalActions/RegularGoalActions.tsx:11]: remove duplicate import of displayToast

**Review by vinaybadgujar102 — CHANGES_REQUESTED** [review]

File: src/components/GoalsComponents/MyGoalActions/RegularGoalActions.tsx (hunk 1 above)
Comment (harshalranjhani) [reply src/components/GoalsComponents/MyGoalActions/RegularGoalActions.tsx]: fixed.

**Review by vinaybadgujar102 — APPROVED** [review]

**Comment by vinaybadgujar102** [comment]
@harshalranjhani Thanks for contributing! We appreciate your work and feel free to contribute more!

**Pull request merged**